pvlib/spectrum/mismatch.py

```diff
@@ -571,3 +571,127 @@ def spectral_factor_caballero(precipitable_water, airmass_absolute, aod500,
     )
     modifier = f_AM + f_AOD + f_PW  # Eq 5
     return modifier
+
+
+def spectral_factor_pelland(airmass_absolute, clearsky_index,
+                            module_type=None, coefficients=None):
+    r"""
+    Estimate a technology-specific spectral mismatch modifier from absolute
+    airmass and clear sky index using the Pelland model.
+
+    The motivation for this model is to include the effects of cloud cover on
+    the spectrum, and thus spectral mismatch factor estimation. Another
+    motivation is to develop a simple parameterisation compared with existing
+    models. Model coefficients are derived using spectral irradiance and other
+    meteorological data from eight locations. These coefficients for seven
+    modules, available here via the ``module_type`` parameter, as well as more
+    details on the model, can be found in [1]_.
+
+    Parameters
+    ----------
+    airmass_absolute : numeric
+        absolute (pressure-adjusted) airmass. [unitless]
+
+    clearsky_index: numeric
+        clear sky index. [unitless]
+
+    module_type : str, optional
+        One of the following PV technology strings from [1]_:
+
+        * ``'fs4-1'`` - First Solar series 4-1 and earlier CdTe module.
+        * ``'fs4-2'`` - First Solar 4-2 and earlier CdTe module.
+        * ``'monosi'``, - anonymous sc-si module.
+        * ``'multisi'``, - anonymous mc-si- module.
+        * ``'cigs'`` - anonymous copper indium gallium selenide module.
+        * ``'asi'`` - anonymous amorphous silicon module.
+
+    coefficients : array-like, optional
+        user-defined coefficients, if not using one of the default coefficient
+        sets via the ``module_type`` parameter.
+
+    Returns
+    -------
+    mismatch: numeric
+        spectral mismatch factor (unitless) which is multiplied
+        with broadband irradiance reaching a module's cells to estimate
+        effective irradiance, i.e., the irradiance that is converted to
+        electrical current.
+
+    Notes
+    -----
+    The Pelland model parameterises the spectral mismatch factor as a function
+    of absolute air mass and the clear sky index as follows:
+
+    .. math::
+
+       M = a_1 K_c^{a_2} AM_a^{a_3},
+
+    where M is the spectral mismatch factor, and :math:`a_1, a_2, a_3` are
+    module-specific coefficients. In the PVSPEC model publication, absolute air
+    mass is estimated using the Kasten and Young model [2]_. The clear sky
```

Comment:
```suggestion
    mass is estimated starting from the Kasten and Young relative air mass [2]_. The clear sky
```

I think many users are not savvy to the difference between absolute and relative air mass, so we should be careful to make it clear that K&Y on its own is not enough.  Feel free to suggest a clearer alternative :)

Reply:
Good point